src/reddit.ts

```diff
@@ -27,46 +28,47 @@ export default class Reddit {
     isInit = true;
   }
 
-  public static async getNotifications(subreddit: string, usernames: string[], game: Game,
+  public static async getNotifications(subreddit: string, users: RedditUserProvider[], game: Game,
                                        date?: Date, limit?: number): Promise<BotNotification[]> {
-    return this.getNotificationsFromSnoowrap(subreddit, usernames, game, date, limit);
+    return await this.getNotificationsFromSnoowrap(subreddit, users, game, date, limit);
```

Comment:
the await here is unnecessary

Reply:
Why is that? Because it is also an `async` fucntion?